### ISSUE TITLE
[Feature Request] Use the Steam Linux Runtime with Winetricks

### ISSUE THREAD

**Issue by sonic2kk** [sonic2kk/steamtinkerlaunch#860]
## System Information
- SteamTinkerLaunch version: git
- Distribution: Arch Linux

## Feature Description
Various Winetricks, especially on SteamOS, require the Steam Linux Runtime to install properly. For example. `dotnetdesktop6`. The Steam Linux Runtime is fetched by SteamTinkerLaunch when requested (such as for a game launch) and we fetch it based on the version specified in a Proton version's `toolmanifest.vdf` file, under its `require_app_id`.

To run Winetricks in the Steam Linux Runtime, the Winetricks command needs to be wrapped in the Steam Linux Runtime command. We have a function that can get the relevant SLR for a given Proton version, and we can check if we have the SLR with `-z ${SLRCMD[@]}`.

The problem with using the SLR with Winetricks is not so straightforward to fix. There are two problem areas:
1. We can't use system-wide Winetricks. The SLR is a container, meaning its scope is very limited. We would need to use a locally-downloaded Winetricks script. STL already allows users to do this, and it will download Winetricks to `~/.config/steamtinkerlaunch/downloads/winetricks/winetricks` (the path will be different on Flatpak). So if the user has Winetricks already downloaded (such as on Steam Deck where it's always preferred even over an unwise system-wide installation) that's fine
2. We need a way to know when to use the SLR with Winetricks. This should be a checkbox, but whether or not it should go on the Winetricks GUI or the Global Menu GUI is something I'm not sure of. Furthermore, we need a way to tell users when the SLR is being used (perhaps in a notification). If the SLR is not found, this will already be managed by `setSLRReap`, the function that gets the SLR for us.

In short, we need to find a way to implement this feature in a way that makes sense for users who already have Winetricks installed system-wide, since we can't use their system-wide installation.

Using the SLR (or more specifically, the ***correct*** SLR) with Winetricks will greatly improve compatibility. It is also another step in the adventure that is adding the SLR to more places around STL. Proton is not intended to be used without the SLR anyway, so this change is sensible and should happen.

Side Note: An update to the Steam Linux Runtime wiki page should be done probably before any work is done on this, to properly document how the SLR works now and how SteamTinkerLaunch is able to fetch it.

**Comment by sonic2kk** [comment]
The go-forward for this should be to have this option on the Global Menu only, and it should only apply if Download Winetricks is also enabled.

Or, perhaps it could enable "Download Winetricks" if enabled, or act in place of it. Not sure, will need a little bit of consideration and likely some messing around in an implementation.
